packages/client/src/route/customRoutes.tsx

```diff
@@ -44,8 +44,12 @@ const CustomRoutes = ({ customRoutes }) => {
     <Suspense fallback={<LoadingCircle message={t('common:loader.loadingCustom')} />}>
       <Routes>
         {matchedRoutes.map((route, i) => {
-          const { route: r, component, props: p } = route
+          let { route: r, component, props: p } = route
           const Element = component as any
+          // e.g. /places/* will get truncated to /places, /places* will get truncated to /places
```

Comment:
I wonder if line 56 will have any impact with this?

as its looking for `r.split('/')[1]` and what if the path in project is like `/dashboard/*`

![image](https://github.com/EtherealEngine/etherealengine/assets/10975502/a6e5f285-32eb-4b55-9d37-7eee32f50310)

Reply:
@barankyle or what the way forward as @kattsushi is waiting for this PR to fix routing on MT.

Reply:
I fixed some issues with this yesterday, it should be working. Just want to make sure the new route schema is what we want.